pkg/cli/cluster_free_disk_space.go

```diff
@@ -0,0 +1,125 @@
+package cli
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+	v1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+
+	rookcli "github.com/rook/rook/pkg/client/clientset/versioned"
+
+	clusterspace "github.com/replicatedhq/kurl/pkg/cluster/space"
+)
+
+const (
+	IsDefaultStorageClassAnnotation = "storageclass.kubernetes.io/is-default-class"
+	OpenEBSLocalProvisioner         = "openebs.io/local"
+	RookRBDProvisioner              = "rook-ceph.rbd.csi.ceph.com"
+	RookCephFSProvisioner           = "rook-ceph.cephfs.csi.ceph.com"
+)
+
+func NewClusterFreeDiskSpaceCmd(cli CLI) *cobra.Command {
+	var forStorageClass, openEBSImage, openEBSNode string
+	logbuf := bytes.NewBuffer(nil)
+	logger := log.New(logbuf, "", 0)
+	k8sConfig := config.GetConfigOrDie()
+	clientSet := kubernetes.NewForConfigOrDie(k8sConfig)
+	rookClientSet := rookcli.NewForConfigOrDie(k8sConfig)
+
+	cmd := &cobra.Command{
+		Use:          "free-disk-space",
+		Short:        "Returns the amount of free disk space (in bytes) for a given storage class",
+		SilenceUsage: true,
```

Comment:
Why are using `SilenceUsage `?

Reply:
Copy and Pasta. What are you worried about ?

Reply:
If we want to not show the usage when we did not use the flags accordingly or an error occurs that is fine 👍

Reply:
That is fine, it is in use in other commands of the same binary.